src/App.tsx

```diff
@@ -216,7 +178,7 @@ function AppContent() {
       }
     };
     populatePlantingSites();
-  }, [dispatch, selectedOrganization.id, activeLocale]);
+  }, [selectedOrganization.id, dispatch, activeLocale]);
```

Comment:
is this a preferred order? properties first then objects..?

Reply:
That is strange... this was auto populated by my IDE but I was not aware that it had any sort of preference.

Reply:
I put it back